project4/style.css

```diff
@@ -0,0 +1,185 @@
+/* == Base styles == */
+* {
+    font-family: Helvetica, sans-serif;
```

Comment:
Ideal font stack would also cover Windows systems which don't usually have Helvetica on them. I know this is from my CodePen, but -- like may bits of demo code -- it's not meant to be perfectly production-ready code.